websites/D/Discords.com/presence.ts

```diff
@@ -3,351 +3,508 @@ const presence = new Presence({
 				},
 			];
 		}
-	} else if (document.location.pathname.includes("/upvote")) {
-		const serverNameUpvote = document.querySelector(
-			"h1.JoinUpvote_clickable-server-name__3l6Es"
+	} else if (pathname.includes("/bio/p")) {
+		presenceData.details = document.querySelector("h1.HiVipESs")?.textContent;
+		presenceData.state = document.querySelector(
+			"div.fHOTdnFx.g0pusrry"
 		)?.textContent;
-		presenceData.details = `Upvoting 🗳️ ${serverNameUpvote || "N/A"} `;
-		if (serverNameUpvote) {
+		presenceData.largeImageKey =
+			document.querySelector("div.jYifcv4i")?.querySelectorAll("img")[1]?.src ??
+			Assets.Logo;
+		presenceData.smallImageKey = Assets.Heart;
+		presenceData.smallImageText =
+			document.querySelector("#like-button")?.textContent === "0"
+				? "Not Liked"
+				: document.querySelector("#like-button")?.textContent;
+		presenceData.buttons = [
+			{
+				label: "View Profile",
+				url: href,
+			},
+		];
+	} else if (pathname.includes("/bots/verification")) {
+		presenceData.details = "🤖 Viewing: Verification";
+		presenceData.buttons = [
+			{
+				label: "View Page",
+				url: href,
+			},
+		];
+
+		if (pathname.includes("/apply")) {
+			presenceData.details = "🤖 Applying for Verification";
 			presenceData.buttons = [
 				{
-					label: `Upvote ${serverNameUpvote}`,
-					url: document.location.href,
+					label: "Apply for Verification",
+					url: href,
 				},
 			];
 		}
-	} else if (document.location.pathname.includes("/servers/")) {
-		presenceData.details = `Viewing 🎨 ${document
-			.querySelector("h1.servernameh1")
-			?.textContent.replace("PREMIUM", " ")}`;
-		presenceData.state = `${
+	} else if (pathname.includes("/bots/search")) {
+		presenceData.details = `🔍 Searching for: ${pathname.split("/search/")[1]}`;
+		if (pageNumberBots) presenceData.state = `📖 Page ${pageNumberBots}`;
+		presenceData.smallImageKey =
+			"https://cdn-icons-png.flaticon.com/512/2021/2021646.png";
+		presenceData.smallImageText = `Found ${
 			document
-				.querySelectorAll("span.mr-2")[0]
-				?.textContent.replace(
-					`${document.querySelectorAll("span.mr-2")[0]?.textContent}`,
-					`${`👥 ${document.querySelectorAll("span.mr-2")[0]?.textContent}`}`
-				) || "0 members online"
-		}, ${
-			document
-				.querySelectorAll("span.mr-2")[1]
-				?.textContent.replace(
-					`${document.querySelectorAll("span.mr-2")[1]?.textContent}`,
-					`${`🎁 ${document.querySelectorAll("span.mr-2")[1]?.textContent}`}`
-				) || "0 members"
-		}, ${
-			document
-				.querySelectorAll("span.mr-2")[2]
-				?.textContent.replace(
-					`${document.querySelectorAll("span.mr-2")[2]?.textContent}`,
-					`${`💎 ${document.querySelectorAll("span.mr-2")[2]?.textContent}`}`
-				) || "0 boosts"
+				.querySelector("div.banner-left-hp")
+				?.querySelector("h1")
+				?.textContent?.split("you")[1]
 		}`;
 		presenceData.buttons = [
 			{
-				label: "View Page",
-				url: document.location.href,
+				label: "View Results",
+				url: href,
 			},
 		];
```

Comment:
Generally speaking, we commend the use of buttons when it links to a specific page instead of 'randomly'. In an example, we don't like having buttons for search results, but are fine with profiles, blog points etc

Reply:
Will remove.